Documentation/gettingstarted/kind.rst

```diff
@@ -54,9 +54,19 @@ Then, install Cilium release via Helm:
 
 .. note::
 
-   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v1
-   controllers ``net_cls`` and ``net_prio`` have to be disabled, or cgroup v1 has
-   to be disabled (e.g. by setting the kernel ``cgroup_no_v1="all"`` parameter).
+   To fully enable Cilium's kube-proxy replacement (:ref:`kubeproxy-free`), cgroup v2
+   needs to be enabled, and cgroup v1 controllers ``net_cls`` and ``net_prio``
```

Comment:
I wonder if we should split the sentence, because it's not entirely clear if it's `(cgroup v2 ... and cgroup v1 controllers ...) or cgroup v1 ...)` or `cgroup v2 ... and (cgroup v1 controllers ... or cgroup v1 ...)`.
Maybe:
```suggestion
   needs to be enabled. Furthermore, cgroup v1 controllers ``net_cls`` and ``net_prio``
```

Reply:
Sure. I just realized that I forgot to add instructions to enable cgroup v2.